_data/hosting_providers.json

```diff
@@ -2448,5 +2448,15 @@
     "plan": "",
     "reviewed": "2020.09.28",
     "note": "Let's Encrypt available on all hosting, VPS, and dedicated server plans."
-  }
+  },
+  {
+    "name": "celltek",
+    "link": "https://celltek-server.de/",
+    "category": "full",
```

Comment:
For "full" support which we've defined [here](https://certbot.eff.org/hosting_providers/), HTTPS needs to be enabled by default. Do you have any documentation to support this claim?

Depending on what you have, it may make sense to add the link to the announcement or plan fields.

Reply:
We use Plesk for end customers with the Lets Encrypt extension, for dedicated servers we use directly the Certbot. 
https://www.plesk.com/extensions/letsencrypt/
https://certbot.eff.org/

Reply:
As described at https://github.com/certbot/website#hosting-providers-list, in order for us to accept this PR you have to provide evidence of the HTTPS support you offer. Do you have an announcement, tutorial, or plan description that you can link to?